wp-content/themes/workingnyc/views/home.twig

```diff
@@ -2,12 +2,12 @@
 
 {% block content %}
 <div class="bg-scale-1">
-  <header class="o-banner {% if post.thumbnail %}bg-banner-min-height bg-shape-wnyc-header{% endif %} relative">
-    <div class="page-max desktop:px-6">
+  <header class="o-banner bg-gradient-to-r from-[#E5F4FF] to-[#fefefe] {% if post.thumbnail %}bg-banner-min-height bg-shape-wnyc-header{% endif %} relative">
+    <div class="page-max desktop:px-26">
 
       {% if post.thumbnail %}
       <div class="py-2 tablet:py-8 desktop:px-6 tablet:grid grid-cols-2 gap-x-5">
```

Comment:
- [ ] This div has two grid columns, and the header text is in the first column. The image should be placed in the second column.